<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemPage</class>
 <widget class="QWidget" name="SystemPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="CustomImageLabel" name="buttonDateTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateTimeText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Date &amp;&amp; Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonUpgrade">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUpgradeText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Firmware
Upgrade</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonLicensePlate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLicensePlateText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>License
Plate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonSecurity">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSecurityText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Security</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonFactoryDefault">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFactoryDefaultText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Factory
Default</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonInformation">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformationText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonConfig">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelConfigText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>160</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonLanguage">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>210</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLanguageText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>360</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>410</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCloseText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <zorder>labelLicensePlateText</zorder>
  <zorder>labelCloseText</zorder>
  <zorder>labelInformationText</zorder>
  <zorder>labelDateTimeText</zorder>
  <zorder>labelFactoryDefaultText</zorder>
  <zorder>labelLanguageText</zorder>
  <zorder>labelSecurityText</zorder>
  <zorder>labelConfigText</zorder>
  <zorder>labelUpgradeText</zorder>
  <zorder>buttonClose</zorder>
  <zorder>buttonFactoryDefault</zorder>
  <zorder>buttonUpgrade</zorder>
  <zorder>buttonInformation</zorder>
  <zorder>buttonConfig</zorder>
  <zorder>buttonLicensePlate</zorder>
  <zorder>buttonDateTime</zorder>
  <zorder>buttonLanguage</zorder>
  <zorder>buttonSecurity</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomImageLabel</class>
   <extends>QLabel</extends>
   <header>ui/customwidget/customimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
